src/MAUI/Maui.Samples/Samples/Layers/AddCustomDynamicEntityDataSource/readme.md

```diff
@@ -0,0 +1,50 @@
+# Add custom dynamic entity data source
+
+Display data from a custom dynamic entity data source using a dynamic entity layer.
+
+![Image of add custom dynamic entity data source](addcustomdynamicentitydatasource.jpg)
+
+## Use case
+
+Using this API developers can connect their own feed as a custom `DynamicEntityDataSource` to leverage observations using a `DynamicEntityLayer`. An example of this is in a mobile situational awareness app where a custom `DynamicEntityDataSource` can connect to peer-to-peer feeds to visualize real time location tracks.
```

Comment:
"Using this API" seems redundant, I think it would read better if it were removed.